egs2/README.md

```diff
@@ -6,93 +6,94 @@ See: https://espnet.github.io/espnet/espnet2_tutorial.html#recipes-using-espnet2
 
 ## Overview of example information
 
-| Directory name          | Corpus name                                                                             | Task                    | Language              | URL                                                                                                          | Note         |
```

Comment:
I do not find a reason why these previous entries need to be changed. Could you try to remain them un-modified?

Reply:
@ftshijt I am not sure why it shows that the previous entries have changed. I have inserted a row for the new recipe based on alphabetical order and put in the details. 
Is there a particular way to update the README.md file?

Thanks!